tests/api2/test_003_network.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+
+# Author: Eric Turgeon
+# License: BSD
+
+import pytest
+import sys
+import os
+
+apifolder = os.getcwd()
+sys.path.append(apifolder)
+from auto_config import ha, user, password, ip
+from functions import GET, PUT, SSH_TEST
+from config import *
+
+if ha and "domain" in os.environ:
+    domain = os.environ["domain"]
+    gateway = os.environ["gateway"]
+    hostname = os.environ["hostname"]
+    hostname_b = os.environ["hostname_b"]
+    primary_dns = os.environ["primary_dns"]
+    secondary_dns = os.environ["secondary_dns"]
+else:
+    from auto_config import hostname, domain
+
+Reason = "BRIDGEDNS is missing in ixautomation.conf"
+dns_cfg = pytest.mark.skipif("BRIDGEDNS" not in locals(), reason=Reason)
+
+
+@pytest.mark.skipif(not ha and "domain" not in os.environ, reason="Skiping test for Core")
+def test_01_set_network_for_ha():
+    payload = {
+        "domain": domain,
+        "ipv4gateway": gateway,
+        "hostname": hostname,
+        "hostname_b": hostname_b,
+        "nameserver1": primary_dns,
+        "nameserver2": secondary_dns
+    }
+    results = PUT("/network/configuration/", payload)
+    assert results.status_code == 200, results.text
+
+
+@pytest.mark.skipif(not ha and "domain" not in os.environ, reason="Skiping test for Core")
+def test_02_force_fenced():
+    cmd = 'fenced --force'
```

Comment:
Running `fenced --force` will cause the other controller to watchdog panic itself.
1. Are we making sure that before we run this command, it's being run from the ACTIVE controller?
2. Or are we expecting to run this on the standby controller so that it causes the active controller to panic and cause a failover event?

Either way, we need to understand why we're running this command because `fenced --force` will always return successful unless there is some underlying OS problem.

Reply:
It is needed for the ha bhyve test

Reply:
What is it testing? `fenced` get calls automatically on the ACTIVE controller.

Reply:
It is not a real test I was asked to add that in the provision.py I am just moving everything in the API test.

Reply:
@yocalebo I had @ericbsd add fenced --force so we could make the active controller grab the disks without having to shut down or remove the passive controller, or disable failover during initial setup.

Reply:
Ran this by William as well "i am indifferent about it" therefore resolving this one also.